test/check-session

```diff
@@ -54,13 +56,13 @@ class TestSession(MachineCase):
         b.click('#login-button')
         b.expect_reload()
         b.wait_page("server")
```

Comment:
I don't think we make any guarantees here.  Also, cockpit-ws and logind don't synchronize with each other, so there is always a little race, I guess.